mne_bids/path.py

```diff
@@ -345,6 +375,23 @@ def copy(self):
         """
         return deepcopy(self)
 
+    def mkdir(self, exist_ok=True):
+        """Create the BIDS path directory.
+
+        Parameters
+        ----------
+        exist_ok : bool
+            Whether it is okay that the directory
+            already exists (default is True).
```

Comment:
```suggestion
        exist_ok : bool
            If ``False``, raise an exception if the directory already exists.
            Otherwise, do nothing (default).
```